homeassistant/components/modbus/__init__.py

```diff
@@ -3,7 +3,7 @@
 
 from typing import Any
 
-import voluptuous as vol
+import voluptuous as vol  # type: ignore
```

Comment:
Why is this needed?

Reply:
this import got flagged when I ran mypy directly on the command line, I have later seen that the githook does not flag it so it is removed.

Reply:
more correctly do not run mypy in the component, it needs to be run from core.